its/src/test/java/com/sonar/it/csharp/Tests.java

```diff
@@ -56,7 +56,7 @@
   DoNotAnalyzeTestFilesTest.class,
   MetricsTest.class,
   NoSonarTest.class,
-  UnitTestResultsTest.class
```

Comment:
Is it normal to have removed one test class?

Reply:
Good catch, thanks!